pkg/nvmeclient/nvme_client.go

```diff
@@ -466,6 +470,7 @@ func Connect(request *ConnectRequest) (*CtrlIdentifier, error) {
 		}
 	}
 	request.Traddr = traddr
+	request.Hostid = "8c3eedd2-46bd-4d06-82d5-a59248990a40" // TODO: implement managed hostid
```

Comment:
Really? This is quite ugly... Why not generate a proper `hostid` where we generate the `hostnqn` and track them together? They should be used as a pair and have the same liftime, so this seems like the obvious approach.

Reply:
You are right @muli. @guy and I talked about it today and I told guy that this is the quickest way to work around the issue. hostid is meaningless without persistent reservations and there is currently no support for it (nor enforcement on hostid uniqueness).

If it is not urgent to work around while we have a proper fix, we can do it the right (compliant) way.

BTW, the hostid can be also a function of the hostnqn. Like a hash of the hostnqn string to a hex string which in turn is converted to a uuid. That is also a way we can avoid managing its lifetime.

Reply:
Thanks @sagi-lb . @guy-lb let's do something that is a little bit more future proof. Either @sagi's suggestion of keying the hostid from the hostnqn or just generating and tracking both at the same time.

Reply:
@sagi-lb @muliby-lb fixed. please review

Reply:
You tagged the wrong Muli, it seems like.

Reply:
@muli those are the risks of having a great github user :) (Mulis FTW!)